tests/shared/models/selection-state.spec.js

```diff
@@ -225,6 +225,18 @@ describe('Model: selectionState', function () {
         m.chai.expect(selectionState.getSelectedDevices()).to.deep.equal([])
       })
     })
+
+    describe('.deselectOtherDrives()', function () {
+      it('should deselect other drives', function () {
+        selectionState.deselectOtherDrives('/dev/disk1')
+        m.chai.expect(selectionState.getSelectedDevices()).to.not.include.members([ '/dev/disk2' ])
+      })
+
+      it('should not remove the specified drive', function () {
+        selectionState.deselectOtherDrives('/dev/sdb')
+        m.chai.expect(selectionState.getSelectedDevices()).to.deep.equal([ this.drives[0].device ])
```

Comment:
You've already got `/dev/sdb` on the line above, so it might be more logical for this line to say:
```
m.chai.expect(selectionState.getSelectedDevices()).to.deep.equal([ '/dev/sdb' ])
```
?

Reply:
I've opted for the other route as it looks more consistent with the tests above – use `this.drives` more.

Reply:
Sweet. And I've just realised that `this.drives` has both `/dev/sdb` (i.e. Linux-style device strings) and `/dev/disk2` (i.e. OSX-style device strings) which is slightly confusing! :laughing: (but that's obviously outside the scope of this PR)

Reply:
Or maybe it _is_ within the scope if this PR? I was only looking at a single commit before and didn't realise that this PR was adding the entirety of the `this.drives` object.